lib/node/fullnode.js

```diff
@@ -404,8 +417,12 @@ class FullNode extends Node {
    */
 
   async getCoinsByAddress(addrs) {
-    const mempool = this.mempool.getCoinsByAddress(addrs);
+    if (!this.mempoolIndex)
```

Comment:
Why return early here when there is no `mempoolIndex`? Can't there be coins that are in the chain db still?